crates/pallets/starknet/src/tests/invoke_tx.rs

```diff
@@ -56,17 +55,23 @@ fn given_hardcoded_contract_run_invoke_tx_then_it_works() {
 
         let transaction: InvokeTransaction = get_invoke_dummy(Felt252Wrapper::ZERO).into();
 
-        let tx = Message {
-            topics: vec![
-                "0xdb80dd488acf86d17c747445b0eabb5d57c541d3bd7b6b87af987858e5066b2b".to_owned(),
-                "0x0000000000000000000000000000000000000000000000000000000000000001".to_owned(),
-                "0x0000000000000000000000000000000000000000000000000000000000000001".to_owned(),
-                "0x01310e2c127c3b511c5ac0fd7949d544bb4d75b8bc83aaeb357e712ecf582771".to_owned(),
+        let tx = HandleL1MessageTransaction {
```

Comment:
Maybe the name of this tests is not as explicit as it should be. Why does it involve consuming a l1 message when it's about `invoke` txs?

Reply:
I have no idea why it was implemented this way but I can remove consuming part.